Sources/PT.PM.CSharpParseTreeUst/Converter/ExpressionVisitor.cs

```diff
@@ -73,6 +73,41 @@ public override Ust VisitAnonymousObjectCreationExpression(AnonymousObjectCreati
             return result;
         }
 
+        public override Ust VisitTupleExpression(TupleExpressionSyntax node)
+        {
+            var result = new TupleCreateExpression()
+            {
+                TextSpan = node.GetTextSpan(),
+                Initializers = new List<Expression>()
+            };
+
+            for (int i = 0; i < node.Arguments.Count; i++)
+            {
+                var arg = node.Arguments[i];
+                var assignment = new AssignmentExpression
+                {
+                    Right = (Expression)VisitAndReturnNullIfError(arg.Expression),
+                    TextSpan = arg.GetTextSpan()
+                };
+
+                assignment.Left = new IdToken($"Item{i + 1}", assignment.TextSpan);
+                result.Initializers.Add(assignment);
+
+                if (arg.NameColon != null)
+                {
+                    var name = (Expression)VisitAndReturnNullIfError(arg.NameColon.Name);
+                    result.Initializers.Add(new AssignmentExpression
+                    {
+                        Left = name,
+                        Right = new MemberReferenceExpression(null, new IdToken($"Item{i + 1}", name.TextSpan), name.TextSpan),
```

Comment:
Is it necessary to create new `IdToken`? Can't we reuse `assignment.Left`?

Besides, I don't see the point of `MemberReferenceExpression` with `null` target, can't we just reuse `assignment.Right` here too? Like this:
```
result.Initializers.Add(new AssignmentExpression
{
    Left = name,
    Right = assignment.Right,
    TextSpan = assignment.TextSpan
}
```

Reply:
yes, it necessary) all tokens should be unique

Reply:
Then at least put that comment regarding `null` in the code. :wink: 

But I still think it's better to just use `assignment.Right` instead.